erpnext_shipping/erpnext_shipping/doctype/letmeship/letmeship.py

```diff
@@ -323,6 +326,10 @@ def get_pickup_delivery_info(self, address, contact):
 			"email": contact.email_id,
 		}
 
+	def check_weight(self, service_response):
+		messages = service_response.get("messages", [])
+		return any("maximum weight" in message.lower() for message in messages)
```

Comment:
Please add a docstring here, to explain what this does. As a reader, I'm not aware what the `message` might contain. Is it `"Maximum weight exceeded."`, `"Maximum weight: 5kg"`, something else...?